recipe/meta.yaml

```diff
@@ -58,7 +58,7 @@ test:
 
 about:
   home: https://www.generic-mapping-tools.org
-  license: LGPL-3.0
+  license: LGPL-3.0-only
```

Comment:
@leouieda and @seisman can you confirm this please?

Reply:
I think it should be `LGPL-3.0-or-later`. See
https://github.com/GenericMappingTools/gmt/blob/master/LICENSE.TXT